src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ResourcesChangeInfo.cs

```diff
@@ -144,6 +144,16 @@ internal static ResourcesChangeInfo CatastrophicDictionaryChangeInfo
             }
         }
 
+        internal IEnumerable<ResourceDictionary> OldDictionaries
+        {
+            get { return _oldDictionaries; }
+        }
+
+        internal IEnumerable<ResourceDictionary> NewDictionaries
+        {
+            get { return _newDictionaries; }
+        }
+
```

Comment:
Are these needed in this PR? I don't see them being utilized anywhere.

Reply:
Removed these